inc/generation.class.php

```diff
@@ -170,6 +155,40 @@ static function showMassiveActionsSubForm(MassiveAction $ma) {
       return true; 
    }
 
+
+   /**
+    * Generate numbers from a massive update
+    *
+    * @since 0.85+1.1
```

Comment:
To be consistent with https://github.com/pluginsGLPI/geninventorynumber/pull/8/files#diff-53f390d375398624afe1cfe1125f42bfR26 :)